### PR TITLE
Add admin login

### PR DESCRIPTION
**Pull request by emilypatsko**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+DATABASE_URL="Server=localhost;port=5432;Database=WhaleSpottingDb;user id=postgres;password=techswitch;"
```
Comment (ChristianMcc) [inline .env]: We don't want to commit this :) it should be covered by the gitignore! If it isn't it's worth checking the gitignore is correct

File: package.json [hunk 2]
```diff
@@ -8,5 +8,16 @@
   },
   "keywords": [],
   "author": "",
+<<<<<<< HEAD
```
Comment (ChristianMcc) [inline package.json]: Ah I must have misread the git gui terminal before (sorry I should have asked for it to be zoomed in!) can you revert any changes to this file and delete the package-lock.json file which is at the root of the repo please? :)

Comment (emilypatsko) [reply package.json]: Think I've done this now!

**Review by ChristianMcc — APPROVED** [review]
LGTM :)

**Pull request merged**